security-policies/bundle/compliance/cis_azure/rules/cis_4_3_8/test.rego

```diff
@@ -0,0 +1,32 @@
+package compliance.cis_azure.rules.cis_4_3_8
+
+import data.cis_azure.test_data
+import data.compliance.policy.azure.data_adapter
+import data.lib.test
+import future.keywords.if
+
+test_violation if {
+	# fail if ssl_enforcement is disabled
+	eval_fail with input as test_data.generate_postgresql_server_with_infrastructure_encryption("Disabled")
```

Comment:
is there an option for empty value? what would be the result in that case?

Reply:
An empty value is different than "Enabled", so it would be considered disabled, which is fine

Reply:
Added a test confirming what I explained https://github.com/elastic/cloudbeat/pull/1797/files#diff-98815038831f8bde93f4e3d0ccf4f4d1a57f715feb0273b9f43e1f989628213dR10